source/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 project(PasswordManager)
 
-add_executable(passwordManager main.cpp PasswordManager.cpp)
+add_library(sqlite3 STATIC ../lib/sqlite3.c)
 
-add_subdirectory(test)
\ No newline at end of file
+add_executable(passwordManager main.cpp PasswordManager.cpp DataBase/DataBaseHandler.cpp DataBase/DataBaseCommon.cpp)
```

Comment:
Add new CMakeLists to the DataBase folder instead of providing relative source paths here.
Try this: https://stackoverflow.com/questions/8934295/add-source-in-a-subdirectory-to-a-cmake-project